.editorconfig

```diff
@@ -17,3 +17,6 @@ trim_trailing_whitespace = false
 [Makefile]
 indent_style = tab
 indent_size = 4
+
+[*.go]
+indent_style = tab
```

Comment:
Does the Go language force the developer to use the `tab`? If it's not a must, and using `space` is not an anti-pattern, I'd like to suggest that let's keep it simple to use spaces, because not all editors have the same indent number configuration of the tab, and to be consistent through the languages.

BTW: Using `tab` in Makefile because it is a must. (or Makefile will fail).

Reply:
`Go` 官方建议使用 `tab` 进行缩进，为了正确的引导大家都遵循这个建议，在核心命令中提供了 `go fmt` 命令，以增加代码可读性，为了更方便的理解，我找到了一些讨论这个话题的链接：

- https://golang.org/doc/effective_go.html#formatting

上面的官方文档中注明:

> We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

早期的 `gofmt` 命令中也曾允许设置缩进风格的选项，可是后来的迭代中，官方直接移除了相关选项，以以避免社区在这个话题上的分化。

- https://www.v2ex.com/t/344953 

V2EX 上面在早期也曾有过这样的讨论。

-----------------------------------

`go fmt` [有着更多的好处，包括缩进，注释排版，代码块分隔等，在风格一致性上具有很好的帮助， #31 中 `.github/workflows/pr.yml` 文件有相关的命令以帮助执行 格式化。

我建议我们在 `go-wechaty` 中，采用 `go fmt` 进行代码格式化。

> 在我们内部团队中，还有另外一些风格指导，比如 `goimport` 我们可以逐步的去引入。

Reply:
让我们尽快合并进去可好